Formula/tt.rb

```diff
@@ -0,0 +1,26 @@
+class Tt < Formula
+  desc "Command-line utility to manage Tarantool applications"
+  homepage "https://github.com/tarantool/tt"
+  url "https://github.com/tarantool/tt/releases/download/v1.0.0/tt-1.0.0-complete.tar.gz"
+  sha256 "6f42d30b9d9f9fbad1907a49cf3394f71c2c86e3faa1194fd9372e4d8877c792"
+  license "BSD-2-Clause"
+
+  depends_on "go" => :build
+  depends_on "mage" => :build
+  depends_on "pkg-config" => :build
+  depends_on "openssl@3"
+  depends_on "wget"
```

Comment:
Since they seem interchangeable this is better
```suggestion
  uses_from_macos "curl"
```

Reply:
Thx, changed